scout/server/blueprints/institutes/controllers.py

```diff
@@ -680,11 +679,47 @@ def cases(store: MongoAdapter, request: request, institute_id: str):
 
     case_groups = get_and_set_cases_by_status(store, request, institute_obj, all_cases, data)
 
-    # Compile the final data
     data["cases"] = [(status, case_groups[status]) for status in CASE_STATUSES]
     return data
 
 
+def case_has_rna(case: dict) -> bool:
+    """Return True/False depending on whether the case has RNA-related data."""
```

Comment:
I'd prefer describing the three different test classes here rather than adding comments for them.